### ISSUE TITLE
api page missing an explanation on date ranges

### ISSUE THREAD

**Issue by aaronkaplan** [cybergreen-net/stats-new#242]
The query interface for the API is great! Did not expect that.

Minor issues:

* [x] describe that risk id -1 means apparently "all risks".
* [ ] I am missing one feature from it: please deescribe that you can also give date ranges (at least it seems so that it's possible!). For example `2015-01-01;2017-01-31` . At least the site (or my browser?) auto-suggests this . However it does not seem to work on the ranks api endpoint. It would make sense to get the time series of ranks over time for each country. Very useful for doing graphs.

**Comment by zelima** [comment]
@aaronkaplan for now we only support fixed date ranges - week, month, quarter and year. It is not possible to set custom range (Eg: 2 month). 
So for now only query parameter `date` is available and it is depended on `granularity` parameter (which by default is month). It is automatically calculated by the given granularity and set to the first date of that range. Eg:
`https://cybergreen-staging.herokuapp.com/api/v1/rankings?granularity=month&risk=1&date=2017-01-25`
and 
`https://cybergreen-staging.herokuapp.com/api/v1/rankings?granularity=month&risk=1&date=2017-01-07`

Will return same result - Country rankings in January of 2017

I will update definitions for both to make this clear.

**Comment by zelima** [comment]
Updated with new explanations.
Also removed `default -1` and included it into description. As think it looks better.
@aaronkaplan is it any better? if yes we can close

**Comment by zelima** [comment]
Closing for now. reopen if needed

**Comment by aaronkaplan** [comment]
nope, I am still missing info on what is allowed to enter. Reopening.

**Comment by rufuspollock** [comment]
@aaronkaplan can we clarify: the issue says "api page missing an explanation on date ranges". Checking the site that is clearly there afaict.

However, you also say:

> However it does not seem to work on the ranks api endpoint. It would make sense to get the time series of ranks over time for each country. Very useful for doing graphs.

This is a feature request for an additional option on the ranks API (as i understand it). If so, I think this is a separate issue "Support data ranges in rank API". If so, let's close this issue as done and open the other issue.

Please can you clarify for us.
